bin/mycroft-help

```diff
@@ -25,6 +25,8 @@ echo "Mycroft-specific commands you can use from the Linux command prompt:"
 echo "  mycroft-cli-client       command line client, useful for debugging"
 echo "  mycroft-msm              Mycroft Skills Manager, to manage your Skills"
 echo "  mycroft-msk              Mycroft Skills Kit, create and share Skills"
+echo "  mycroft-start            Launch/restart Mycroft services"
```

Comment:
I'm not sure about this help line. from reading it I would expect that entering

`mycroft-start`

would start Mycroft. but I could just be reading to much into it. When launching the command it's pretty obvious what's needed to run it.

Reply:
I'm open to better phrasing.  I expected if someone types it they'll figure it out pretty easily.